Source/Scene/Cesium3DTileset.js

```diff
@@ -787,6 +799,7 @@ define([
         var length = children.length;
         for (var i = 0; i < length; ++i) {
             var child = children[i];
+            child.updateTransform(); // Need to update transform and bounding volumes before checking distance
```

Comment:
I would prefer not to put the `updateTransform` call here but otherwise we'll just have another small function that loops over all the children and only updates transforms. Maybe that's ok...